src/core_plugins/kibana/server/lib/export/collect_dashboards.js

```diff
@@ -0,0 +1,21 @@
+import collectPanels from './collect_panels';
+
+export const deps = {
```

Comment:
What's up with this pattern? Why do you re-export functions from sibling modules under the `deps` object?